Absolute Imports Not Working in Javascript Project

### What is the problem?

As per title. 

### Steps to Reproduce: <!-- if you can, link to a repo with the failing code -->

1. Create a Javascript project with `blitz new app --js`
2. Create any component
3. Import it in a page

### Versions:

```
debug: local
debug: pkgPath: /home/jfelix/Documents/programming/open-source/test-m2/node_modules/@blitzjs/cli 

Linux 5.3 | linux-x64 | Node: v12.16.1

blitz: 0.8.2 (global)
blitz: 0.8.2 (local)
```

### Supporting Documentation

Error Log: 

```
./pages/_app.js
Module not found: Can't resolve 'app/styles/theme' in '/home/jfelix/Documents/programming/open-source/test-m2/.blitz/caches/dev/pages'
Error: Cannot find module '/home/jfelix/Documents/programming/open-source/test-m2/.blitz/caches/dev/.next/server/static/development/pages/_document'
Require stack:
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/load-components.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/api-utils.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/server/next.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/server/lib/start-server.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/cli/next-dev.js
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/bin/next
- /home/jfelix/Documents/programming/open-source/test-m2/node_modules/@blitzjs/server/bin/next-patched
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:982:15)
    at Function.Module._load (internal/modules/cjs/loader.js:864:27)
    at Module.require (internal/modules/cjs/loader.js:1044:19)
    at require (internal/modules/cjs/helpers.js:77:18)
    at Object.loadComponents (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/load-components.js:24:25)
    at DevServer.findPageComponents (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:555:60)
    at DevServer.renderErrorToHTML (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:852:33)
    at DevServer.renderErrorToHTML (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/server/next-dev-server.js:19:915)
    at async DevServer.render (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:540:22)
    at async Object.fn (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:390:17)
    at async Router.execute (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/router.js:134:32)
    at async DevServer.run (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:505:29)
    at async DevServer.handleRequest (/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js:149:20) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/load-components.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/api-utils.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/next-server/server/next-server.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/server/next.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/server/lib/start-server.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/cli/next-dev.js',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/next/dist/bin/next',
    '/home/jfelix/Documents/programming/open-source/test-m2/node_modules/@blitzjs/server/bin/next-patched'
  ]
}
```

Compiled code:

![image](https://user-images.githubusercontent.com/21092519/81013485-30b20d80-8e29-11ea-86f6-b32062050fc8.png)

--- a/packages/generator/src/generators/app-generator.ts
+++ b/packages/generator/src/generators/app-generator.ts
@@ -23,7 +23,7 @@ export class AppGenerator extends Generator<AppGeneratorOptions> {
     if (!this.options.useTs) {
       return ['tsconfig.json']
     }
-    return []
+    return ['jsconfig.json']
   }
 
   async getTemplateValues() {

--- a/packages/generator/templates/app/jsconfig.json
+++ b/packages/generator/templates/app/jsconfig.json
@@ -1,0 +1,5 @@
+{
+  "compilerOptions": {
+    "baseUrl": "."
+  }
+}

